backend/routes/api/users.js

```diff
@@ -83,12 +109,30 @@ router.get("/", async (req, res) => {
 //     }
 // );
 
+// Delete a user
+router.delete(
+    '/:id',
+    async (req, res) => {
+        const user = await User.findByPk(req.params.id);
+        if (!user) {
+            res.status(404);
+            return res.json({ message: 'User not found' });
+        }
+        await user.destroy();
+        return res.status(201).json({ message: 'User successfully deleted' })
+    })
+
```

Comment:
Are you checking if the person sending this delete request is properly authorized?